src/services/authentication/strategies/TSPStrategy.js

```diff
@@ -0,0 +1,90 @@
+const { AuthenticationBaseStrategy } = require('@feathersjs/authentication');
+const { NotAuthenticated } = require('@feathersjs/errors');
+
+const logger = require('../../../logger');
+const {
+	decryptToken, verifyToken,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['verifyToken' is assigned a value but never used. (no-unused-vars)](https://app.codacy.com/app/schul-cloud/schulcloud-server/pullRequest?prid=4605061)